app/scripts/lib/get-first-preferred-lang-code.js

```diff
@@ -18,7 +17,14 @@ const existingLocaleCodes = allLocales.map(locale => locale.code.toLowerCase().r
  *
  */
 async function getFirstPreferredLangCode () {
-  const userPreferredLocaleCodes = await getPreferredLocales()
+  let userPreferredLocaleCodes = await getPreferredLocales()
+  
+  // safeguard for Brave Browser until they implement chrome.i18n.getAcceptLanguages
+  // https://github.com/MetaMask/metamask-extension/issues/4270
+  if(!userPreferredLocaleCodes){
```

Comment:
Can I catch it? Not really; it's not raised by your code, but the cleanup code that handles async tasks in Chrome API; therefore using try ... catch in your callback won't help. Since it's asynchronous, using try around the original API call won't help either.

shoutouts to: https://stackoverflow.com/questions/28431505/unchecked-runtime-lasterror-when-using-chrome-api